### ISSUE TITLE
Performance Issue

### ISSUE THREAD

**Issue by wsjyhaozi** [ish-app/ish#880]
Hello I was suggested to open a issue from your twitter. 
So tried to compile the same C++ code on my macbookpro and iSH. And it turns out the speed to run an executable it's 10-500x slower. 
If the code is very simple the difference could be only 10x. For example this one:
```
#include <iostream>
#include <cmath>
#include <ctime>
int main() {
    int size = 1000000;
    double array[size];

    std::clock_t t1 = std::clock();
    for (int i = 0; i < size; ++i) {
        array[i] = std::rand();
    }
    std::clock_t t2 = std::clock();
    std::cout << float(t2 - t1) / CLOCKS_PER_SEC << "\n";

    return 0;
}
```

And I complied it through `g++ -O3 -o run.o test.cpp` on both devices. On my macbookpro "run.o" prints 0.01, while on iSH(ipad pro) it prints 0.12. Jut 10x slower. 

I have another much more complicated code related with my work so I can't show you here... There are lots of headers, classes, loops etc..And I complied exactly the same way as above. it takes 0.06s on my laptop to run but 32s on iSH. 500x slower! I thought it was because iSH is an emulator but you think it could be potentially optimized?

**Comment by tbodt** [comment]
Yeah with simple programs, the slowdown is much less. Would you be able to post the program that has a 500x slowdown so I can look into that?

**Comment by wsjyhaozi** [comment]
Unfortunately I can't show you the code at current stage because it's about a not yet published academic paper... But I'll try to write a new code which includes all numerical stuff I used in the code. I'll post it here once done!

**Comment by econcz** [comment]
I’m having trouble with python wheel builds. For example, “pip3 -vvv install statsmodels” requires numpy 1.7.5 which has to be built (no corresponding version of py3-numpy in repositories) and hangs for hours without completion.
![C12C9CEC-6F00-454C-B378-BFA95450627F](https://user-images.githubusercontent.com/29724411/92091597-47657280-edd1-11ea-9030-c90468a699d9.png)
PS statsmodels website says it is buildable on i386.

**Comment by Assfugil** [comment]
Plug it in and wait a few weeks.
